traffic_mirroring.go

```diff
@@ -30,36 +25,36 @@ func main() {
 	}
 
 	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
-	max_size := 10
-	packets_list := make([][]byte,max_size) 
-	current_packet := 0
+	packetsList := make(chan []byte, 22)
+	filePath := "Test"
+	go writeFile(packetsList, filePath)
 
 	for overlayPacket := range packetSource.Packets() {
 		vxlanLayer := overlayPacket.Layer(layers.LayerTypeVXLAN)
 		if vxlanLayer != nil {
 			vxlanPacket, _ := vxlanLayer.(*layers.VXLAN)
-			
-			fmt.Println(current_packet)
-			packets_list[current_packet] = vxlanPacket.LayerPayload()
-			current_packet += 1
+			packetsList <- vxlanPacket.LayerPayload()
 			fmt.Println(vxlanPacket.LayerPayload())
```

Comment:
This `fmt.Println` is here for test reasons, right? Please, unit test this code and remove such print. The performance can be degraded by such writes in the filesystem.